### PR TITLE
feat(create-release): share a semantic-release config that honours `type!:`

### LINKED ISSUE #653: semantic-release: bang-style breaking commits (`feat!:`) produce no release at all
> 🤖 Generated by the Daily AI Engineer

## Evidence

While wiring `create-release.yaml` into `devantler-tech/world-at-ruin`, I measured what the
portfolio's standard `.releaserc` actually does with a bang-style breaking commit. Tested with
`semantic-release@25.0.3` (the exact version `create-release.yaml` pins) against a clean baseline —
a local tag at `HEAD`, so only the test commit is analysed:

| commit | bare `.releaserc` | expected |
|---|---|---|
| `feat!: …` | **no release at all** | major |
| `fix!: …` | **no release at all** | major |
| `feat(scope)!: …` | **no release at all** | major |
| `feat: …` | minor | minor |
| `fix: …` | patch | patch |
| `docs:` / `chore:` / `ci:` | no release | no release |
| `feat:` + `BREAKING CHANGE:` footer | major | major |

The failure mode is not a wrong bump — a bang-style breaking change produces **no release
whatsoever**, so it ships silently unversioned and consumers never learn the API broke.

Cause: `@semantic-release/commit-analyzer` defaults to the **Angular** preset, which recognises
only the `BREAKING CHANGE:` footer, not the Conventional Commits `!` shorthand.

## Why this is portfolio-wide, not a world-at-ruin quirk

Every `.releaserc` in the org is the bare form (`ksail`, `go-template`, `dotnet-template`,
`platform`, `platform-template`, `gitops-tenant-template`, `aws`, `.github`, `wedding-app`,
`ascoachingogvaner`, and `actions`' own test fixture) — some with extra `releaseRules`, none
configuring the bang.

Meanwhile the **org commit-message ruleset explicitly permits it**:

```
^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\([\w\-\.]+\))?(!)?:
                                                                                ^^^^^
```

So the org's own rules invite `feat!:`, and the release pipeline then silently drops it. Any repo
here is one bang-commit away from an unreleased breaking change.

## Hypothesis

Fixing this centrally in `create-release.yaml` removes the trap for every consumer at once, rather
than requiring each repo to discover it independently.

## Proposed direction

Two options, and I'd like a steer:

1. **Install the preset in the shared workflow** — add
   `conventional-changelog-conventionalcommits` to the `npx` invocation so consumers can set
   `"preset": "conventionalcommits"`, which handles `!` natively. Verified it is **not** currently
   available: with the bare `npx semantic-release@25.0.3`, that preset fails with
   `Cannot find module 'conventional-changelog-conventionalcommits'`. Cleanest semantically, but
   changes the shared workflow's dependency surface for everyone.
2. **Document the per-repo `.releaserc` fix** — `parserOpts.breakingHeaderPattern` plus a
   `breaking: true` release rule, which works with only bundled dependencies. This is what
   world-at-ruin now ships (devantler-tech/world-at-ruin#108); verified across all eight commit
   forms above, with the `BREAKING CHANGE:` footer path unaffected.

Option 1 is the better fix if the added dependency is acceptable; option 2 is already proven and
needs no change here.

## Acceptance criteria

- [ ] `feat!:` / `fix!:` / `feat(scope)!:` produce a **major** release on a repo using the shared
      workflow.
- [ ] `feat:`, `fix:`, `docs:`, `chore:`, `ci:` and the `BREAKING CHANGE:` footer path are
      **unchanged** (verified, not assumed — this is the regression risk).
- [ ] Existing consumers are migrated or explicitly confirmed unaffected.

## Size

Small-to-medium — the change is tiny; the care is in not regressing bump behaviour across ~10 repos.

### PR DESCRIPTION
**Pull request by devantler**
> 🤖 Generated by the Daily AI Engineer

## Why

Every repo using this workflow has a bare `.releaserc`, and a bare `.releaserc` gives a bang-style
breaking commit — `feat!:` — **no release at all**. Not a wrong version: no release, so a breaking
change ships silently unversioned and nobody downstream learns the API broke.

The org's own commit-message rule explicitly permits the bang, so contributors are invited to write
exactly the form that silently does nothing. This affects ksail, platform, the templates and the
rest — any of them is one such commit away from an unreleased breaking change.

## What

Fixes it once, here, instead of in ten repos. The workflow now ships a shared release configuration
that a repo can point its `.releaserc` at, so future changes to release policy also happen in one
place rather than being copied around.

Opt-in and additive — repos that don't adopt it behave exactly as before.

⚠️ **Adopting repos must drop their own `plugins` list** when they switch to the shared config; a
local list silently overrides the shared one and keeps the old broken behaviour. I'd propagate that
to consumers as a follow-up sweep once you're happy with this.

Fixes #653

### CODE REVIEW

**Comment by devantler** [comment]
> 🤖 Generated by the Daily AI Engineer

**Validation record.** This is a load-bearing shared workflow with ~10 consumers, so everything
below was measured rather than reasoned about.

**The bug, measured** (semantic-release 25.0.3, clean baseline — a local tag at HEAD so only the
commit under test is analysed; without that, commits already on the branch since the last tag
contaminate every case):

```
                                bare config    with shared config
feat!: / fix!: / feat(scope)!:   no release  ->  major release
feat:                            minor       ->  minor
fix:                             patch       ->  patch
docs: / chore: / ci:             no release  ->  no release
feat: + BREAKING CHANGE footer   major       ->  major
```

Only the bang forms change; nothing else moves.

**Design decisions, each verified rather than assumed:**

1. **`extends` merge semantics.** Tested all three shapes: `{extends, branches}` with no local
   `plugins` → **major** ✅; bare config → no release; `extends` **plus** a local `plugins` array →
   **no release**, because the local array *replaces* the shared one. That is why adopting repos
   must drop `plugins`, and why this is opt-in rather than automatic.
2. **`conventionalcommits` preset rejected.** It handles `!` natively but is **not bundled** —
   verified `Cannot find module 'conventional-changelog-conventionalcommits'` against a real
   install. Adopting it would mean adding a dependency to this workflow for every consumer.
   `parserOpts.breakingHeaderPattern` + a `breaking: true` rule gets the same result with bundled
   deps only.
3. **Inline heredoc rather than reading `release-config/` from this repo.** A reusable workflow has
   the *consumer's* repo checked out, so it cannot read its own files without a second checkout at a
   ref it cannot reliably derive (`job_workflow_sha` is documented ambiguously enough that I did not
   want to depend on it). Inlining pins the config to the very commit consumers already pin by SHA.
   Extracted the step's `run` with `yq` and executed it for real — it produces files **byte-identical**
   to `release-config/`.
4. **Deep-path `extends`** (`.../tag-only.json`) works, so one package serves both the tag-only
   repos (ksail, go-template) and the full-release repos.

**The new self-test is RED/GREEN proven on both axes** — a test that cannot fail is worthless:

- *Behaviour:* stripped the bang fix from the canonical file **and** the heredoc, so drift stayed
  green and only behaviour could fail → `bump mismatch for 'feat!: breaking': expected 'major
  release', got 'no release'`. That is the original bug, reproduced by the guard.
- *Drift:* edited only the canonical copy → `release-config/index.json has drifted from the heredoc`.
- Restored → both green.

It asserts the **bump**, not the config's shape, because a shape assertion would not have caught
this bug.

**Compatibility:** the existing `test-create-release-config.sh` asserts the exact `run:` string of
the 🎉 Release step; that step is untouched, and the new work is a separate preceding step.

**No lint regression:** `actionlint` findings on `ci.yaml` are the 4 pre-existing `code-quality`
scope warnings (lines 414/1154/1775/1788 — none mine); `zizmor --persona=pedantic` on
`create-release.yaml` is identical to baseline (2 `template-injection` helps, both pre-existing).

**Not verifiable from here:** the workflow running in a real consumer's CI. This repo's own
`test-create-release` dry-run jobs exercise `create-release.yaml` on every PR, so that lands with
this change.

@codex review — particularly whether the inline heredoc can produce malformed JSON under any
consumer's runner shell, whether `npm`/`node_modules` in the consumer workspace could collide with a
consumer that has its own `node_modules`, and whether the new step could break a consumer that does
*not* use `extends`.

File: release-config/index.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      {
+        "parserOpts": {
+          "breakingHeaderPattern": "^(\\w*)(?:\\(([\\w\\-\\.]*)\\))?!: (.*)$"
+        },
+        "releaseRules": [
+          { "breaking": true, "release": "major" }
+        ]
+      }
+    ],
+    "@semantic-release/release-notes-generator",
```
[the remark this answers is not in the document]
Comment (devantler) [reply release-config/index.json]: > 🤖 Generated by the Daily AI Engineer

All four findings fixed in `54fdb7b`, plus a fifth your review led me to.

**1. Release notes not marking bang commits as breaking** — correct and the most consequential of the four. `parserOpts` was on `commit-analyzer` only, so a `feat!:` would bump major while the published notes said nothing about the break — losing precisely the warning this change exists to surface. Both plugins now share the pattern.

**2. Scope charset too narrow** — correct. `feat(api/client)!:` matched neither my override nor the fallback, so it silently produced no major release: the original bug, in a narrower case. Widened to `(.*)`, with a test asserting your exact example.

**3. Guard not load-bearing** — correct, and the sharpest one. A test that can fail without blocking the required check is decoration. Added to both `ci-required-checks.needs` and the aggregate's `job-results`; this repo's own parity check now reports *"needs ↔ job-results are in sync"*. (This is the third time in my recent work that I built a guard which could pass or fail without consequence — noted.)

**4. Root layout invariant** — correct, and I should have caught it: `AGENTS.md` states the root holds only composite-action folders plus dotfiles, *specifically* so nothing is mistaken for an action. Moved to `.github/release-config/`.

**5. The heredoc was unnecessary — found while fixing #4.** Reading `AGENTS.md` properly, this repo already documents a **same-commit self-checkout** pattern (`job.workflow_repository` @ `job.workflow_sha`, in use by `update-agent-skills.yaml`) for exactly the problem I worked around. My PR body claimed the ref "cannot be reliably derived"; that was wrong — the repo had solved it and I hadn't read far enough.

Switching to it **removes the embedded duplicate entirely**. There is now one copy of the config, so the drift half of the test is obsolete and deleted. The test is smaller and the design conforms to the repo's stated convention instead of inventing a parallel one.

*(`actionlint` flags `job.workflow_repository`/`job.workflow_sha` as unknown — it flags the existing `update-agent-skills.yaml` usage identically, 4 occurrences, and actionlint is not part of this repo's CI. Pre-existing tool gap, not new.)*

**The CI failure on the previous head is also addressed diagnostically:** the test passed locally and failed in CI reporting only `expected X, got <none>`, which is not enough to debug from a log. It now prints semantic-release's actual output on mismatch. Re-verified green locally under CI-like conditions (no `GITHUB_TOKEN`), and RED/GREEN re-proven: removing the bang fix reproduces `expected 'major release', got 'no release'`.

**Comment by devantler** [comment]
> 🤖 Generated by the Daily AI Engineer

@codex review — all four findings addressed at `54fdb7b`, plus a fifth your #4 led me to: this repo already documents a same-commit self-checkout pattern (`job.workflow_repository` @ `job.workflow_sha`) for a reusable workflow reading its own files, which I had worked around with an inline heredoc. Switched to it, so the embedded duplicate and its drift test are gone entirely. Re-requesting at the new head.

File: .github/workflows/create-release.yaml [hunk 2]
```diff
@@ -47,6 +47,37 @@ jobs:
...
+          path: .devantler-tech-actions
+          sparse-checkout: .github/release-config
+          persist-credentials: false
+
+      - name: 🧩 Install the shared release config
+        run: |
+          set -euo pipefail
+          dir=node_modules/@devantler-tech/release-config
+          mkdir -p "$dir"
+          cp .devantler-tech-actions/.github/release-config/*.json "$dir/"
```
[the remark this answers is not in the document]
Comment (devantler) [reply .github/workflows/create-release.yaml]: > 🤖 Generated by the Daily AI Engineer

All three fixed in `a69e7f7`. The P1 is the important one — **it invalidated my earlier validation, not just the code.**

**P1 — MODULE_NOT_FOUND.** You were right. I built a production-shaped fixture (npx, *no* local semantic-release, shared config in workspace `node_modules`) and reproduced it:

```
code: 'MODULE_NOT_FOUND'
```

My bump test had passed only because its fixture ran `npm install semantic-release` — which the real workflow does not. So every "proven" result I reported earlier was proven against the wrong shape. That is a worse error than the bug itself.

Confirmed the repair in the same fixture: a workspace install makes the *unchanged* `npx semantic-release@25.0.3` command resolve the extended config → `major release`. The workflow now installs it when the flag is on, `--no-save` with a scratch manifest if the caller has no `package.json`, so a caller's manifest is never mutated.

**P2 — ran for every caller.** Also right, and my "fully backward compatible / unaffected" claim in the PR body was simply false: a self-checkout auth or fetch hiccup would have aborted releases for repos that never opted in. Now behind `use-shared-config`, **default-off**, per this repo's own feature-flag-first rule — which you correctly cited and I had not applied.

**P2 — missing `contents: read`.** Right; the job inherited the workflow's `permissions: {}`. Added.

**Plus the CI failure you'd have seen on the last head.** The diagnostics I added paid for themselves immediately: semantic-release's CI detection reads `GITHUB_REF`, so the fixture was analysing `refs/pull/655/merge` instead of its own branch — which is exactly why it passed locally and failed in CI. The ambient Actions environment is now stripped for the fixture run, verified by re-running with `GITHUB_REF`/`GITHUB_ACTIONS`/`CI` set.

**Coverage for the gap itself:** new `[Test] Create Release - Shared Config Opt-In` job runs the opt-in path through the *real* workflow, which is the coverage whose absence let MODULE_NOT_FOUND reach review. Wired into `ci-required-checks`; the repo's parity check reports needs ↔ job-results in sync.

**Comment by devantler** [comment]
> 🤖 Generated by the Daily AI Engineer

@codex review — all three addressed at `a69e7f7`. The shared-config path is now opt-in (`use-shared-config`, default-off), the job has `contents: read`, and semantic-release is installed into the workspace so an extended config actually resolves (your MODULE_NOT_FOUND was reproduced in a production-shaped fixture and is now covered by a new CI job). Re-requesting at the new head.

**Comment by devantler** [comment]
> 🤖 Generated by the Daily AI Engineer

## Redesigned — the zizmor finding was the signal the shape was wrong

`zizmor` flagged the `npm install` as an unpinned ad-hoc package fetch, and it was right. Rather than
suppress it, I looked at *why* the install was needed at all — and the answer indicted the whole
approach.

The `extends` / shared-package route had accumulated, cumulatively:

- consumers must delete their `plugins` key (a local array replaces the shared one),
- a same-commit self-checkout,
- `contents: read`,
- an `npm install` into the caller's workspace — because an extended config resolves plugins against
  the workspace while the Release step runs `npx`, which installs into the npx cache
  (`MODULE_NOT_FOUND`),
- …and that install is precisely what zizmor flagged.

Each fix was individually correct and the pile kept growing. That is usually a sign the shape is
wrong, not that one more patch is needed.

## What it does now

Injects the bang pattern into the caller's **existing** `.releaserc` in the ephemeral workspace.
Nothing is committed. Consequences:

| | extends approach | injection |
|---|---|---|
| consumer `.releaserc` change | **required** | **none** |
| `npm install` (zizmor finding) | required | **none** |
| self-checkout + `contents: read` | required | **none** |
| plugin resolution | broken under npx | **unchanged from today** |

`zizmor` is back to the **baseline 2 findings**. Consumers adopt it with one workflow input and no
config edit — which is what "central" should have meant from the start.

## Verified against every real consumer shape

Ran the extracted step against the actual config shapes in this org — tag-only (ksail, go-template),
full (dotnet-template), custom `releaseRules` (**this repo**), and `draftRelease` (world-at-ruin).
**Caller settings win**: `parserOpts`/`releaseRules` are merged, never replaced, so this repo's
`ci` → patch rule survives, a caller's own `breakingHeaderPattern` is respected, and `github`'s
`draftRelease` is untouched.

## The test now proves the bug and the fix in one run

Production-shaped (`npx`, nothing installed into the workspace) with a control case:

```
inject=no   feat!: breaking   -> no release      <- the bug
inject=yes  feat!: breaking   -> major release   <- the fix
inject=yes  fix!: / feat(scope)!: / feat(api/client)!:  -> major release
inject=yes  feat: / fix: / docs: / chore: / ci:          -> unchanged
```

The earlier version installed semantic-release locally and so passed while the real workflow was
broken — the failure you identified. It also strips the ambient Actions environment, since
semantic-release reads `GITHUB_REF` even under `--no-ci` (the cause of the CI-vs-local discrepancy).

@codex review — particularly the `jq` merge semantics (can a caller's config shape make it drop or
duplicate a plugin?), whether a non-JSON `.releaserc` is handled safely, and whether mutating the
workspace config could surprise a later step.

File: .github/workflows/create-release.yaml [hunk 3]
```diff
@@ -52,6 +57,56 @@ jobs:
...
+              then [., {parserOpts: bang, releaseRules: [{breaking: true, release: "major"}]}]
+              elif type == "string" and . == "@semantic-release/release-notes-generator"
+              then [., {parserOpts: bang}]
+              elif type == "array" and .[0] == "@semantic-release/commit-analyzer"
+              then [.[0], ((.[1] // {}) + {parserOpts: (bang + ((.[1].parserOpts) // {})),
+                           releaseRules: (((.[1].releaseRules) // []) + [{breaking: true, release: "major"}])})]
+              elif type == "array" and .[0] == "@semantic-release/release-notes-generator"
+              then [.[0], ((.[1] // {}) + {parserOpts: (bang + ((.[1].parserOpts) // {}))})]
+              else . end)
+          ' "$config" > "$config.bang" && mv "$config.bang" "$config"
```
[the remark this answers is not in the document]
Comment (devantler) [reply .github/workflows/create-release.yaml]: > 🤖 Generated by the Daily AI Engineer

All three fixed in `62399e4`. The first is the one that matters.

**1. The rewrite failed OPEN — and I reproduced it.**

```
$ printf '{"branches":["main"]}' > .releaserc      # valid config, no plugins key
$ jq '.plugins |= map(.)' .releaserc > out && mv out .releaserc
jq: error (at .releaserc:0): Cannot iterate over null
```

On the left of `&&` a failure is a *tested* condition, so `set -e` does not abort, and the trailing `echo` then made the step report success with the config untouched. `breaking-bang-commits: true` would have silently left `feat!:` unreleased — **the exact failure this feature exists to prevent, reintroduced by its own implementation.**

Now: separate statements (no `&&`), explicit preconditions, and a post-rewrite assertion that the pattern actually landed.

I want to name the pattern rather than just fix it: **this is the fourth vacuous-success bug in my work this session** — after the authenticated GHCR "public" check, the non-blocking reachability gate, and the unwired required check. Same shape every time: a guard that cannot fail when the thing it guards is broken. It is now an explicit item on my pre-ship checklist.

**2. Other config locations** — correct. `package.json`'s `release` key and `.releaserc.json` now work. JS and YAML configs cannot be rewritten safely, so they exit with an error naming the file and the fix — never a silent skip, which would recreate finding #1 in a new place.

**3. README** — correct, and the repo's reusable-workflow convention requires it. Documented with usage, the no-`.releaserc`-change guarantee, and the supported-config constraint.

**Every path exercised, not just the happy one:**

```
MUST SUCCEED   bare .releaserc (ksail) · .releaserc.json · package.json release key
               · full shape (dotnet-template) · custom releaseRules (this repo)   -> all PASS
MUST FAIL      no plugins key · no config · .releaserc.js · YAML in .releaserc     -> all FAIL, distinct messages
```

That sweep also caught a **false negative in my own new assertion**: with a top-level config the verification query interpolated to `..plugins`, a jq syntax error, so a *correct* injection reported failure. Testing only the happy path would have shipped it.

Bump matrix re-verified in production shape with the `inject=no` control still reproducing the bug.

**Comment by devantler** [comment]
> 🤖 Generated by the Daily AI Engineer

@codex review — all three addressed at `62399e4`: the jq rewrite now fails closed (reproduced the fail-open first), `package.json`/`.releaserc.json` are supported while JS/YAML configs error explicitly, and the README documents the input. Every config path exercised, which also caught a false negative in my own assertion. Re-requesting at the new head.

File: .github/workflows/create-release.yaml [hunk 4]
```diff
@@ -52,6 +57,97 @@ jobs:
...
+
+          # semantic-release's own config search order.
+          config=""
+          json_path=""
+          for f in .releaserc .releaserc.json .releaserc.yaml .releaserc.yml \
+                   .releaserc.js .releaserc.cjs .releaserc.mjs \
+                   release.config.js release.config.cjs release.config.mjs; do
+            if [ -f "$f" ]; then config="$f"; break; fi
+          done
+          if [ -z "$config" ] && [ -f package.json ] && jq -e 'has("release")' package.json >/dev/null 2>&1; then
```
[the remark this answers is not in the document]
Comment (devantler) [reply .github/workflows/create-release.yaml:95]: > 🤖 Generated by the Daily AI Engineer

All four confirmed. I tested #3 expecting to refute it and was wrong.

**#3 — releaseRule suppression is overridden. VERIFIED, not conceded.** I assumed first-match-wins would protect a caller's rule since mine is appended last. Built the case:

```
releaseRules: [ {scope: "no-release", release: false},     <- caller's, FIRST
                {breaking: true, release: "major"} ]        <- mine, appended

commit: feat(no-release)!: …     ->  major release
```

The suppression does **not** win. So enabling this option would silently start releasing commits a caller had deliberately excluded — a behaviour regression in exactly the repos careful enough to configure suppressions.

**#1 — config precedence.** Correct: `cosmiconfig("release").search()` checks `package.json` before the rc files, while my loop checks `.releaserc` first. A caller with both would get the rc file edited and `package.json` actually used — the injection silently doing nothing, which is the fail-open class again.

**#2 — default plugin configs.** Correct that `{"branches": ["main"]}` is a valid, common config that I reject. Worth stating why materialising the defaults is *not* a safe fix: semantic-release's default plugin list includes **`@semantic-release/npm`**, so writing it out explicitly could start publishing packages to npm for repos that never intended it. Failing closed is right; the error message should explain the limitation rather than imply the config is invalid.

**#4 — `@semantic-release/git`.** Correct: with config in `package.json`, an in-place edit sits in the working tree and `@semantic-release/git` would commit the injected block into the caller's repo. Nothing in the current design prevents that.

## Stepping back

That is **14 findings across 5 rounds** on this PR, and the last four are all the same shape: the mechanism has to be correct for *every possible consumer configuration* — location precedence, default plugins, arbitrary `releaseRules` interactions, other plugins' side effects. That surface is open-ended, and each round finds another corner of it.

The alternative — adding the bang config to each repo's own `.releaserc` — handles exactly one configuration per repo: the one that repo actually has. No rewriting, no precedence, no interaction effects. It is already running in production in `world-at-ruin`.

I am putting that trade-off to the maintainer rather than opening a sixth round, since "central if possible" was the instruction and the evidence on *possible* has changed materially. Your findings are what produced that evidence — none of this was visible from the design.

**Comment by devantler** [comment]
> 🤖 Generated by the Daily AI Engineer

**Closing in favour of the per-repo fix (maintainer decision, 2026-07-18).**

Not because the goal was wrong — a central fix is genuinely better when it works — but because the
evidence on *whether it works* changed as review progressed. Recording it so this is not
re-attempted blind.

## Why central turned out to be the wrong shape here

Rewriting a consumer's config means being correct for **every possible consumer config**. Five
review rounds surfaced, in order:

1. `extends` does not merge `plugins` — adopters must delete their own key (so not zero-touch after all).
2. `extends` + `npx` → **`MODULE_NOT_FOUND`**: plugins resolve against the workspace, npx installs to
   its cache.
3. Fixing that needs an `npm install` into the caller's workspace — an unpinned ad-hoc fetch, which
   **zizmor flags**.
4. Injecting into the existing config instead avoided all of that — but then: config-location
   precedence (`cosmiconfig` checks `package.json` first), configs relying on default plugins,
   `@semantic-release/git` committing the injected block, and —
5. **a caller's deliberate `releaseRules` suppression being overridden.** Verified:

   ```
   releaseRules: [ {scope: "no-release", release: false},   <- caller's, first
                   {breaking: true, release: "major"} ]      <- injected
   commit: feat(no-release)!:   ->  major release
   ```

   Enabling it would silently start releasing commits a repo had deliberately excluded — a
   behaviour regression precisely in the repos careful enough to configure suppressions.

That is 14 findings across 5 rounds, and the surface is open-ended: each round found another
configuration shape to be correct about.

## What replaces it

The same fix in each repo's own `.releaserc`. It handles exactly one configuration — the one that
repo actually has — so precedence, default plugins, plugin side effects and rule interactions all
stop being variables. It is already running in production in `devantler-tech/world-at-ruin`.

## What was worth keeping

- The **bug** is real and org-wide: a bare `.releaserc` gives `feat!:` **no release at all** (#653
  has the full measurement).
- The **fix content** — `parserOpts.breakingHeaderPattern` + a `breaking: true` rule — is unchanged
  and proven; only the delivery mechanism is dropped.
- The **`conventionalcommits` preset is not usable**: not bundled with semantic-release.
- Anyone re-attempting a central version should start from finding 5, not from the design.

Every finding here came from review, not from the design being obviously wrong up front. Closing
with thanks rather than as a failure.
